src/utils/address_translation.ml

```diff
@@ -2,6 +2,9 @@ open Core_kernel
 open Bap.Std
 
 
+let last_addr = ref None
```

Comment:
`last_addr` should be a local variable of `generate_tid_map`, not a global variable.